interface/forms/soap/C_FormSOAP.class.php

```diff
@@ -1,40 +1,46 @@
 <?php
 
-/**
+/*
  * soap form
- *
  * @package   OpenEMR
  * @link      http://www.open-emr.org
  * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
  * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2022 Sherwin Gaddis <sherwingaddis@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 require_once($GLOBALS['fileroot'] . "/library/forms.inc.php");
 require_once("FormSOAP.class.php");
 
-use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
 
 class C_FormSOAP extends Controller
 {
-    var $template_dir;
-
-    function __construct($template_mod = "general")
+    private TwigContainer $twig;
+    public function __construct()
     {
-        parent::__construct();
-        $this->template_mod = $template_mod;
-        $this->template_dir = dirname(__FILE__) . "/templates/";
-        $this->assign("FORM_ACTION", $GLOBALS['web_root']);
-        $this->assign("DONT_SAVE_LINK", $GLOBALS['form_exit_url']);
-        $this->assign("STYLE", $GLOBALS['style']);
-        $this->assign("CSRF_TOKEN_FORM", CsrfUtils::collectCsrfToken());
+        $path = self::getTemplatePath();
```

Comment:
actually, just realized that using wrong call here (this is for a static function). Should be:
```
$path = $this->getTemplatePath();
```